src/styles/blocks/catalog/catalog.scss

```diff
@@ -0,0 +1,126 @@
+.catalog {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  place-items: center;
+  gap: 46px 48px;
+  grid-auto-rows: 1fr;
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @include on-mobile-l {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include on-mobile-s {
+    grid-template-columns: 1fr;
+  }
+
+  &__card {
+    width: 200px;
+    border: 1px solid $grey-300;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    transition: all 300ms;
```

Comment:
It is recommended to specify the transition-property instead of `all` for better performance

Reply:
@polosanya I will fix it.

just for your info the task was 
"Make all the changes smooth on hover (during 300ms):"

Reply:
@polosanya never mind, I have read the docs.